tasks/main.yml

```diff
@@ -23,6 +23,7 @@
     name: "{{ item }}"
     state: present
   with_items: "{{ ice_devel_packages }}"
+  when: 'ice_install_devel | default(True)'
```

Comment:
Your guess is as good as mine. There isn't a defaults file at the moment which is why I did everything in-line, but happy to change.